gdrive_api_tests/test_1_selenium_go_through_tree.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from handlers.ui_functions import UIFunctions
+
+
+class GDriveTreeTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.ui = UIFunctions()
```

Comment:
UIFunctions are declared in GDriveTreeTest and in LoginTest, so there is code duplication. Please create a TestBase class, create a setUpClass there, define cls.ui = UIFunctions() there and inherit all of your tests from TestBase. :)

Reply:
Is it better to create multiple variations of TestBase class (eg. with and without login/logout) or create one TestBase class with parameters in setUpClass and tearDownClass, which will allow to exclude login/logout actions, if needed?